.github/workflows/static-checks-mypy-docs.yml

```
@@ -96,7 +96,7 @@ jobs:
   static-checks:
     timeout-minutes: 45
     name: "Static checks"
-    runs-on: ${{ fromJSON(inputs.runs-on-as-json-default) }}
+    runs-on: ${{ fromJSON(inputs.runs-on-) }}
```

Seems like an accidental deletion:

```suggestion
    runs-on: ${{ fromJSON(inputs.runs-on-as-json-default) }}
```